src/libp11-int.h

```diff
@@ -336,6 +336,10 @@ extern int pkcs11_generate_key(PKCS11_TOKEN * token,
 	int algorithm, unsigned int bits,
 	char *label, unsigned char* id, size_t id_len);
 
+/* Generate and store a private EC key on the token */
+extern int pkcs11_generate_ec_key(PKCS11_TOKEN * token, const char *curve,
+	char *label, unsigned char* id, size_t id_len);
```

Comment:
A question. We had internal discussion about the constness of *label* and *id*. As the pkcs11_generate_key() did not have the const decided not to add in here. But logically it would make library users life a bit easier if the signature would be:
```c
extern int pkcs11_generate_ec_key(PKCS11_TOKEN * token, const char *curve,
	const char *label, const unsigned char* id, size_t id_len);
```

I understand that in the PKCS#11 world there is no concept of 'const' so from that design point of view it should be as what is now in the PR.